OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -86,7 +87,9 @@ bool WormAttack(Actor worm)
 
 		void PlayAttackAnimation(Actor self)
 		{
-			renderUnit.PlayCustomAnim(self, "sand");
+			foreach (var notify in self.TraitsImplementing<INotifyAttack>())
```

Comment:
I don't think that is a good idea in terms of making the code easier to understand, because the interface is only triggered to play the animation, but okay. Updated.